java/index.shtml

```diff
@@ -97,7 +97,9 @@ Install package 'java-11-openjdk-headless' to provide command 'java'? [N/y]
   <p><code>sudo apt install openjdk-11-jre</code>  Debian based</p>
   <p><code>sudo dnf install java-11-openjdk</code>  Red Hat based</p>
 
-<h2>For 32-bit or 64-bit computers</h2>
+<h2>Java Installers for Windows, macOS or Linux 32-bit or 64-bit computers</h2>
```

Comment:
This probably should indicate that the Linux reference is for those distributions that don't have a package manager that includes Java.